docs/MISSION_AND_VALUES.md

```diff
@@ -0,0 +1,82 @@
+is easy to use and fully installable on Windows, Mac-OS, and Ubuntu.
+* being **well-documented** with our entire API having up-to-date, useful docstrings
+and additionally providing examples of more complex use-cases when possible.
+* providing a **consistent and stable API** to users by following
+[semantic versioning](https://semver.org/) and limiting the amount of breaking changes
+introduced unless necessary for the future robustness or scalability of the library.
+* sustaining **comparable or better performance when compared to more tailored file
+format reading libraries**. We will regularly run benchmarks utilizing a set of varied
+size images from all the file formats the library is capable of reading.
+* **working closely with the microscopy community** while deciding on standards and best
+practices for open, accessible, file formats and imaging and in deciding which
+proprietary file formats and metadata selection are in need of support.
+
+## Values
+* We are **inclusive**. We welcome and mentor newcomers who are making their first
+contribution and strive to grow our most dedicated contributors into core developers. We
+have a [Code of Conduct](./CODE_OF_CONDUCT.md) to ensure that the bioio remains
+a welcoming place for all.
+* We are **community-driven**. We respond to feature requests and proposals on our
+[issue tracker](https://github.com/bioio-devs/bioio/issues) and make
+decisions that are driven by our user's requirements.
+* We focus on **file IO and metadata conversion**, leaving image analysis functionality
+and visualization to other libraries.
+* We aim to **develop new methods of metadata extraction and conversion**, instead of
+duplicating existing, or porting, from other libraries primarily by creating **language
+agnostic methods** for metadata manipulation.
+* We value **simple, readable implementations**. Readable code that is easy to
+understand, for newcomers and maintainers alike, makes it easier to contribute new code
+as well as prevent bugs.
+* We value **education and documentation**. All functions should have docstrings,
+preferably with examples, and major functionality should be explained in our tutorials.
+Core developers can take an active role in finishing documentation examples.
+* We **minimize [magic](https://en.wikipedia.org/wiki/Magic_(programming))** and always
+provide a way for users to opt out of magical behaviour and guessing by providing
+explicit ways to control functionality.
+
+## Acknowledgments
+We share a lot of our mission and values with the `napari` project, and acknowledge the
+influence of their mission and values statement on this document.
+
+Additionally, much of the work produced for this library is built on the shoulders of
+giants. Notably:
+* [Christoph Gohlke](https://www.lfd.uci.edu/~gohlke/) -- maintainer of `tifffile`,
+`czifile`, and the `imagecodecs` libraries
+* [Paul Watkins](https://github.com/elhuhdron) -- original creator of `pylibczi`
```

Comment:
I wonder if the acknowledgement of Paul (and the reference to `czifile` above it) can now be entirely in the bioio-czi package.
Probably the reference to tifffile and imagecodecs could stay here.